templates/layouts/default.pug

```diff
@@ -56,6 +56,20 @@ html(lang='en')
       .topnav_container
         a(href='https://github.com/FAForever')
           .fab.fa-github.fa-lg
+      ul.nav_item.topnav_container
+        li.nav_list
+          .fab.fas.fa-language
+            ul.nav_absolute.navLanguage
```

Comment:
Here you need to add the ?lng= for the links to work as you not need to specify the file locaion